ibm/resource_ibm_cis_firewall_test.go

```diff
@@ -0,0 +1,127 @@
+package ibm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccIBMCisFirewall_Basic(t *testing.T) {
+
+	var record string
+	name := "lockdowns"
+
+	resource.Test(t, resource.TestCase{
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckIBMCisFirewallDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckIBMCisFirewallConfigCisDS_Basic(name),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckIBMCisFirewallExists("ibm_cis_firewall.lockdown", &record),
+					resource.TestCheckResourceAttr(
+						"ibm_cis_firewall.lockdown", "firewall_type", "lockdowns"),
+					resource.TestCheckResourceAttr(
+						"ibm_cis_firewall.lockdown", "lockdown.#", "1"),
+				),
+			},
+		},
+	})
+}
+func TestAccIBMCisFirewall_Import(t *testing.T) {
+
+	var record string
+	name := "lockdowns"
+
+	resource.Test(t, resource.TestCase{
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckIBMCisFirewallDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckIBMCisFirewallConfigCisDS_Basic(name),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckIBMCisFirewallExists("ibm_cis_firewall.lockdown", &record),
+					resource.TestCheckResourceAttr(
+						"ibm_cis_firewall.lockdown", "firewall_type", "lockdowns"),
+					resource.TestCheckResourceAttr(
+						"ibm_cis_firewall.lockdown", "lockdown.#", "1"),
+				),
+			},
```

Comment:
Add one more block to add update scenario also and function testAccCheckIBMCisFirewallConfigCisDS_Update